.devcontainer/configuration.yaml

```diff
@@ -1,8 +1,5 @@
 default_config:
 
-# ffmeg
-ffmpeg:
-
```

Comment:
In my web search I came across some advice that adding this empty `ffmpeg:` block avoided ”some“ `ffmpeg` errors. But like other people, I found that it doesn't avoid any errors, so I suggest deleting it (less is more).